assembl/static2/js/app/components/administration/voteSession/tokensForm.jsx

```diff
@@ -0,0 +1,126 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import { Checkbox, DropdownButton, MenuItem } from 'react-bootstrap';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+import { getEntryValueForLocale } from '../../../utils/i18n';
+import Helper from '../../common/helper';
+import TokenTypeForm from './tokenTypeForm';
+import {
+  updateTokenVoteInstructions,
+  createTokenVoteType,
+  deleteTokenVoteType,
+  updateTokenVoteExclusive
+} from '../../../actions/adminActions/voteSession';
+
+type TokensFormProps = {
+  instructions: string,
+  exclusive: boolean,
+  tokenTypeNumber: number,
+  tokenTypes: Object,
```

Comment:
`tokenTypes` is more likely an `Array` of something (or maybe an `Immutable.List` ?)